### PR TITLE
remove useGnosis flag

### PR DESCRIPTION
**Pull request by andrewdamelio**
- remove useGnosis choice
- consolidate localStorage values for accountType, acccountWallet

### CODE REVIEW

File: packages/augur-ui/src/modules/app/actions/init-augur.ts [hunk 1]
```diff
@@ -69,10 +69,18 @@ function pollForAccount(
...
       attemptedLogin = true;
 
-      const loggedInAccount = windowApp.localStorage.getItem('loggedInAccount');
-      const loggedInAccountType = windowApp.localStorage.getItem(
-        'loggedInAccountType'
-      );
+      let loggedInAccount = null;
+      let loggedInAccountType = null
+
+      try {
```
Comment (pgebheim) [inline packages/augur-ui/src/modules/app/actions/init-augur.ts]: Yay for defensive programming!

File: packages/augur-ui/src/modules/app/actions/init-augur.ts [hunk 2]
```diff
@@ -69,10 +69,18 @@ function pollForAccount(
...
+        const loggedInUser =  JSON.parse(windowApp.localStorage.getItem('loggedInUser'));
+        loggedInAccount = loggedInUser && loggedInUser.address;
```
Comment (pgebheim) [inline packages/augur-ui/src/modules/app/actions/init-augur.ts]: is it "fine" if `loggedInAccount === false` ?

Seems better to test that we have a loggedInUser and then just extract the properties at that time.

File: packages/augur-ui/src/modules/auth/actions/load-account-data.ts [hunk 3]
```diff
@@ -26,8 +26,12 @@ export const loadAccountData = (
     loginAccount &&
     loginAccount.meta
   ) {
-    windowApp.localStorage.setItem('loggedInAccountType', loginAccount.meta.accountType);
-    windowApp.localStorage.setItem('loggedInAccount', address);
+    const loggedInUser = {
+      accountType: loginAccount.meta.accountType,
+      address
+    }
```
Comment (pgebheim) [inline packages/augur-ui/src/modules/auth/actions/load-account-data.ts]:
```suggestion
    };
```

File: packages/augur-ui/src/modules/auth/actions/logout.ts [hunk 4]
```diff
@@ -17,8 +17,7 @@ export function logout() {
     if (localStorageRef && localStorageRef.removeItem) {
       localStorageRef.removeItem('airbitz.current_user');
       localStorageRef.removeItem('airbitz.users');
-      localStorageRef.removeItem('loggedInAccount');
-      localStorageRef.removeItem('loggedInAccountType');
+      localStorageRef.removeItem('loggedInUser');
```
Comment (pgebheim) [inline packages/augur-ui/src/modules/auth/actions/logout.ts:20]: Should we be removing the pending safes here too, or do you need those hanging around?

File: packages/augur-ui/src/modules/auth/actions/update-sdk.ts [hunk 5]
```diff
@@ -20,10 +20,9 @@ export const updateSdk = (
 
   try {
     dispatch(updateAppStatus(Ox_ENABLED, augurSdk.sdk.zeroX));
-
-    // TODO: useGnosis should always be true or fetchable from
     // the SDK -- maybe augurSdk.sdk.getUseGnosisSafe() ??
```
Comment (pgebheim) [inline packages/augur-ui/src/modules/auth/actions/update-sdk.ts]:
```suggestion
```

File: packages/augur-ui/src/modules/auth/actions/update-sdk.ts [hunk 6]
```diff
@@ -20,10 +20,9 @@ export const updateSdk = (
...
-    const useGnosis = window.localStorage.getItem('isGnosis');
+    // Overwrite to disable gnosis
+    const useGnosis = true;
```
Comment (pgebheim) [inline packages/augur-ui/src/modules/auth/actions/update-sdk.ts]: This should really come from the sdk -- we can configure it at the highest level there and pass in that we're gonna use it from the config.

**Review by pgebheim — CHANGES_REQUESTED** [review]
High level note:

I think when using local storage we should take the time to wrap that access in a class or give it some types so that the code is easier to verify.

Instead of having code strewen about in routes, local-account-data actions, etc I would expect to see one service/lib that lets me just load/set the data and handle error cases appropriately.

File: packages/augur-ui/src/modules/auth/actions/logout.ts (hunk 4 above)
Comment (andrewdamelio) [reply packages/augur-ui/src/modules/auth/actions/logout.ts:20]: we should keep them around to restore safes properly for users with many safes.

**Review by ghost — COMMENTED** [review]
This pull request has been automatically deployed to [FeaturePeek](https://dashboard.featurepeek.com). 🤙
Your deployment will be kept up-to-date with this pull request's latest changes.

**https://peek.run/qo0h79k**

-------

*Please read [our docs](https://docs.featurepeek.com/configuring-peek-yml/) for more configuration details.*

**Review by bthaile — COMMENTED** [review]
lgtm, just wanted to comment.

**Comment by andrewdamelio** [comment]
@pgebheim 👀 when you have a chance

**Review by pgebheim — APPROVED** [review]

**Pull request merged**